applications/registrar/src/reg_authn_req.erl

```diff
@@ -502,3 +503,10 @@ maybe_enforce_security({#auth_user{doc=JObj}=User, Acc}) ->
 -spec maybe_set_encryption_flags({auth_user(), kz_term:proplist()}) -> {auth_user(), kz_term:proplist()}.
 maybe_set_encryption_flags({#auth_user{doc=JObj}=User, Acc}) ->
     {User, encryption_method_map(Acc, JObj)}.
+
+-spec maybe_add_hotdesk_current_id(auth_user()) -> kz_term:proplist().
+maybe_add_hotdesk_current_id(#auth_user{doc=JObj}) ->
```

Comment:
Honestly I would just call `hotdesk_ids/1` directly and pattern match:

```erlang
case kzd_devices:hotdesk_ids(JObj, []) of
    [] -> [];
    [Id|_]= -> [{...}]
end.
```

Reply:
Explicit is better than implicit ;)

Reply:
Sorry, I meant you don't need the `is_hotdesked/1` call at all. Just the pattern match on `hotdesk_ids/1` will tell you all you need to know.